sequencer.go

```diff
@@ -108,6 +108,10 @@ func (p Plan) Validate(ctx context.Context, n int, pb ProgressBar) (err error) {
 		in      = make(chan Job)
 		fileMap = make(map[string]*os.File)
 	)
+	if mock := os.Getenv("DESYNC_MOCK_VALID_PLAN"); mock != "" {
```

Comment:
Not a fan of injecting testing variables via the environment. Let's think of alternatives:
1) The test could move into the `desync` package rather than `main`. This would provide more control over what is mocked. One could use an interface with a testing-implementation, or a non-exported global variable that the test sets before running
2) Exporting a variable that can be set from `main`. That's not ideal though.
Any other ideas?

Reply:
Thanks for the suggestions!
I'll research a bit more about the possible solutions and I'll try to improve this PR.

Reply:
After reading the Go documentation and a few articles online, I ended up using a non-exported global variable to flag the mock validation.

Please let me know if you preferred a different approach instead.